src/libraries/System.Net.Sockets/tests/FunctionalTests/TelemetryTest.cs

```diff
@@ -35,16 +34,351 @@ public static void EventSource_ExistsWithCorrectId()
             Assert.NotEmpty(EventSource.GenerateManifest(esType, esType.Assembly.Location));
         }
 
+        public static IEnumerable<object[]> SocketMethods_MemberData()
+        {
+            yield return new[] { "Sync" };
+            yield return new[] { "Task" };
+            yield return new[] { "Apm" };
+            yield return new[] { "Eap" };
```

Comment:
Would be great to refactor this to have a shared enum and the method `GetHelperBase` right in `SocketTestHelper.cs`.
If that's too much chore, I would at least add a comment that these are strings because of `RemoteExecutor`.

Reply:
Looks like this is the only place we are creating these classes directly, I added a comment regarding string usage.